cmd/gossamer/config.go

```diff
@@ -21,12 +21,11 @@ import (
 	"strconv"
 	"strings"
 
+	database "github.com/ChainSafe/chaindb"
```

Comment:
was this intentionally moved?

Reply:
no, maybe my IDE did it when I did goimports? not sure.